src/aat/resources/features/F-108/F-108.feature

```diff
@@ -0,0 +1,268 @@
+    And the request [is configured to search for exact money field value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+    @S-621
+  Scenario: Should return case for exact match on a Number field
+      Given a case that has just been created as in [S-621_Create_Case_Private_Autotest1]
+      And logstash has finished indexing case data
+      And a user with [a valid profile]
+      And the request [is configured to search for exact number field value from previously created case]
+      And a request is prepared with appropriate values
+      When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+      Then the response [contains the previously created case]
+      And the response has all other details as expected
+
+    @S-622
+  Scenario: Should return case for exact match on a PhoneUK field
+    Given a case that has just been created as in [S-622_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
+    And a user with [a valid profile]
+    And the request [is configured to search for exact PhoneUK value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+    @S-623
+  Scenario: Should return case for exact match on a Text Area field
+    Given a case that has just been created as in [S-623_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
+    And a user with [a valid profile]
+    And the request [is configured to search for exact Text Area field value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+
+  @S-624
+  Scenario: Should return case for exact match on a Text field
+    Given a case that has just been created as in [S-624_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
+    And a user with [a valid profile]
+    And the request [is configured to search for exact Text field value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
```

Comment:
in general, could we please use commas at end of all lines, and a full stop at the end of the last line?

Reply:
needs actioning, though a minor comment. @tomxelliott